node-tests/fixtures/helper-test/mocha.js

```diff
@@ -1,5 +1,4 @@
 import { expect } from 'chai';
-
```

Comment:
I've created a second commit which removes the extra newlines from the fixtures. I can squash this into the first one if we want that change.